app/models/concerns/with_qualifications.rb

```diff
@@ -60,12 +60,28 @@ def qualifications
       end
     end
 
+    def full_qualifications
+      case qualification
+      when "qts" then "Qualified teacher status (QTS)"
+      when "pgce_with_qts" then "Postgraduate certificate in education (PGCE) with qualified teacher status (QTS)"
+      when "pgde_with_qts" then "Postgraduate diploma in education (PGDE) with qualified teacher status (QTS)"
+      when "pgce" then "Postgraduate certificate in education (PGCE) without qualified teacher status (QTS)"
+      when "pgde" then "Postgraduate diploma in education (PGDE) without qualified teacher status (QTS)"
+      end
+    end
+
     def qualifications_description
       return "" unless qualifications
 
       qualifications.map(&:upcase).sort.join(" with ")
     end
 
+    def full_qualification_descriptions
+      return "" unless full_qualifications
```

Comment:
```suggestion
      "" || full_qualifications
```

Maybe

Reply:
Had a chat offline, keeping it as is because it's easier on the brain to read.